.circleci/config.yml

```diff
@@ -1,80 +1,109 @@
-          name: Download Dependencies
-          command: ./gradlew androidDependencies
-    - save_cache:
-          paths:
-            - ~/.gradle
-          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
-    - run:
+      - *attach_workspace
+      - run:
           name: Run Tests
           command: ./gradlew lint test
-    - store_artifacts:
+      - store_artifacts:
           path: app/build/reports
           destination: reports
-    - store_test_results:
+      - store_test_results:
           path: app/build/test-results
-    - run:
-          name: Initial build
-          command: ./gradlew clean assembleDebug --no-daemon --stacktrace
-    - store_artifacts:
-          path: app/build/outputs/apk/
-          destination: apks/
-    - run:
-          name: Upload to Slack
-          command: |
-            export GIT_COMMIT_DESC=$(git log --format=oneline -n 1 | sed -E 's/^[^ ]+ (.*)$/\1/g')
-            curl -F file=@app/build/outputs/apk/debug/app-debug.apk -F channels=$SLACK_CHANNEL -F token=$SLACK_API_TOKEN -F title="${CIRCLE_PROJECT_REPONAME} | branch -> ${CIRCLE_BRANCH} | commit -> ${GIT_COMMIT_DESC}"  https://slack.com/api/files.upload
+      - *persist_to_workspace
 
+  debug-build:
+    <<: *config_android
+
+    steps:
+      - *attach_workspace
+      - run:
+            name: Initial debug build
+            command: ./gradlew clean assembleDebug --no-daemon --stacktrace
+      - *store_artifacts
+      - run:
+            name: Uploading debug build to Slack
+            command: |
+              export GIT_COMMIT_DESC=$(git log --format=oneline -n 1 | sed -E 's/^[^ ]+ (.*)$/\1/g')
+              curl -F file=@app/build/outputs/apk/debug/app-debug.apk -F channels=$SLACK_CHANNEL -F token=$SLACK_API_TOKEN -F title="${CIRCLE_PROJECT_REPONAME} | branch -> ${CIRCLE_BRANCH} | commit -> ${GIT_COMMIT_DESC}"  https://slack.com/api/files.upload
+
+  release-build:
+    <<: *config_android
+
+    steps:
+      - *attach_workspace
+      - run:
+            name: Initial release build
+            command: ./gradlew clean assembleRelease --no-daemon --stacktrace
+      - *store_artifacts
+      - run:
+            name: Upload release build to Slack
+            command: |
+              export GIT_COMMIT_DESC=$(git log --format=oneline -n 1 | sed -E 's/^[^ ]+ (.*)$/\1/g')
+              curl -F file=@app/build/outputs/apk/release/app-release.apk -F channels=$SLACK_CHANNEL -F token=$SLACK_API_TOKEN -F title="${CIRCLE_PROJECT_REPONAME} | branch -> ${CIRCLE_BRANCH} | commit -> ${GIT_COMMIT_DESC}"  https://slack.com/api/files.upload
 
 workflows:
- version: 2
- deploy:
-   jobs:
-     - check-build:
-         filters:
-           branches:
-             ignore:
-               - develop+MVP+RX
-               - /^dev-build-.*/
-     - dev-build:
-         filters:
-           branches:
-             only: 
-               - develop+MVP+RX
-               - /^dev-build-.*/
+  version: 2
+  build_workflow:
```

Comment:
Please add the workflows that we are currently using on the develop+MPV+RX branch. (check-build for all branches except develop+MPV+RX and dev-build-* branches and dev-build for those)